d2target/d2target.go

```diff
@@ -293,11 +293,12 @@ type Connection struct {
 	DstArrow Arrowhead `json:"dstArrow"`
 	DstLabel string    `json:"dstLabel"`
 
-	Opacity     float64 `json:"opacity"`
-	StrokeDash  float64 `json:"strokeDash"`
-	StrokeWidth int     `json:"strokeWidth"`
-	Stroke      string  `json:"stroke"`
-	Fill        string  `json:"fill,omitempty"`
+	Opacity      float64 `json:"opacity"`
+	StrokeDash   float64 `json:"strokeDash"`
+	StrokeWidth  int     `json:"strokeWidth"`
+	Stroke       string  `json:"stroke"`
+	Fill         string  `json:"fill,omitempty"`
+	BorderRadius float64 `json:"borderRadius,omitempty"`
```

Comment:
this should be an int. people won't need this much precision for border radius.

Reply:
but finally before render, we need to convert it to floast64 anyway
<img width="1134" alt="image" src="https://user-images.githubusercontent.com/19243114/222047015-78940496-263f-40db-8dae-4b8d9d0505f5.png">

is it necessary to define BorderRadius as an int?

Reply:
ahh, or maybe we can convert `dist/2` to an int

Reply:
no the distance should still be precise i think. in fact that code should probably be `dist/2.0`.

> is it necessary to define BorderRadius as an int?

hmmmm actually this just made me think of something. i was thinking users wouldn't need it, but maybe we can designate 0-1 values as percentages. then `0.5` means 50% border-radius! 

this was one of the things that D2 couldn't do in recreating this diagram: https://github.com/terrastruct/d2/issues/920

want to include that change in here? @donglixiaoche . if not, no worries, can defer to another PR

Reply:
sure, let me think about it

Reply:
i guess it doesn't make sense for connections because they don't have dimensions (https://stackoverflow.com/a/29966500/20901387). would only be applicable for shapes. anyway, still worth keeping as float64

Reply:
okok

Reply:
will track here: https://github.com/terrastruct/d2/issues/947